src/grafana_dashboards/opensearch.json

```diff
@@ -295,7 +295,7 @@
               "value": "null"
             }
           ],
-          "valueName": "avg"
```

Comment:
"avg" means the moving average window across a sample. Issue here is, if one node goes down and up, we will see for a brief time something like "3.89" nodes, instead of 4->3->4.